common/spinner.py

```diff
@@ -6,9 +6,9 @@
 class Spinner:
   def __init__(self):
     try:
-      self.spinner_proc = subprocess.Popen(["./spinner"],
+      self.spinner_proc = subprocess.Popen(["./spinner.py"],
```

Comment:
we don't even need to Popen anymore! we can't remove this wrapping :)

Reply:
y